Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml

```diff
@@ -617,6 +617,12 @@ script:
         To get the ID of the file you want to download, use the msgraph-list-drive-content command.
       name: item_id
       required: true
+    - name: file_name
+      description: |-
+        The MS Graph file name.
+        To get the file name of the file you want to download, use the msgraph-list-drive-content command,
+        if not provided, the file name will be item_id.
```

Comment:
will it be `item_id` or _the_ `item_id` value?

Reply:
item_id value
I changed